persistence.go

```diff
@@ -163,17 +175,16 @@ func (db sqlitePersistence) Chats(from, to int) ([]*Chat, error) {
 			&pkey,
 			&chat.UnviewedMessagesCount,
 			&chat.LastClockValue,
-			&chat.LastMessageContentType,
-			&chat.LastMessageContent,
+			&lastMessageContentType,
+			&lastMessageContent,
 			&encodedMembers,
 			&encodedMembershipUpdates,
 		)
 		if err != nil {
 			return nil, err
 		}
-
-		// Restore the backward compatible ID
-		chat.ID = chat.ID[:len(chat.ID)-2]
+		chat.LastMessageContent = lastMessageContent.String
```

Comment:
It would throw an error,  as strings are not `Nullable`, that's what I saw by using the api, we never had this problem as we always set it to empty string before, but since the field is nullable we should cater for this case